processor/src/main/java/org/mapstruct/ap/internal/util/Executables.java

```diff
@@ -198,16 +195,10 @@ private static void addNotYetOverridden(Elements elementUtils, List<ExecutableEl
         alreadyCollected.addAll( 0, safeToAdd );
     }
 
-    private static boolean isNotStaticMethodInInterface(ExecutableElement element, TypeElement parentType) {
-        return !( parentType.getKind().isInterface() &&
+    public static boolean isInterfaceDefaultMethod(ExecutableElement element, TypeElement parentType) {
```

Comment:
Are there other default methods than "interface default methods"? I.e. is the check for the interface required at all?